### PR TITLE
Officer view cases

### PR DESCRIPTION
**Pull request by theDustRoom**
Changes from PR41 have been rolled back.

### CODE REVIEW

File: app/views/v8/partials/officer-view-nav.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <nav class="moj-sub-navigation" aria-label="Sub navigation">
+      <ul class="moj-sub-navigation__list">
+        <li class="moj-sub-navigation__item">
+          <a class="moj-sub-navigation__link govuk-link--no-visited-state" {% if currentPage === 1 %}aria-current="page"{% endif %} href="{{url}}">Overview</a>
```
Comment (liammcmurray) [inline app/views/v8/partials/officer-view-nav.html:6]: If on 'Active cases' view, clicking the 'Overview' link does not take you back - just reloads current page.
Assume some snipping of {{url}} should happen to remove trailing '/officer-view-cases'.

Question whether the aria-current page should be a link, but this is consistent with MoJ design pattern so will feed back to community.

Clicking 'Active cases' a second time breaks the prototype. We can work around this in testing (and it's unlikely to happen) - but if it's an easy fix...

**Review by liammcmurray — COMMENTED** [review]
Nice and tidy, and breaking the nav out into a partial makes complete sense. Great that the caseload is realistic as well.
Only one comment to address on officer-view-nav partial.

File: app/views/v8/partials/officer-view-nav.html (hunk 1 above)
Comment (theDustRoom) [reply app/views/v8/partials/officer-view-nav.html]: Weird, was sure those links worked correctly. Fixed that now though.
aria-current="page" is correct see: https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaCurrent

**Review by liammcmurray — APPROVED** [review]
Fantastic!
I've seen an error where licence cases aren't being added to the community total but that's an easy fix and not something you missed - I'll tweak that post-merge. Thanks again Paul ⭐

**Pull request merged**